api/v1alpha/src/auth/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+
+/************************************************************
+  API Endpoints
+ ************************************************************/
+// Define the routes for the REST API
+var auth = express.Router();
+// methods that require multiple routes
+
+/**
+ * @swagger
+ *
+ * /projects/{projectId}/auth::custom
```

Comment:
This route should be explicit for the OAS documentation, since when a client tries to test in a OAS UI, the sample curl commands will be '/auth::custom' instead of '/auth:isDataProducer'. I was going to reference https://github.com/GoogleCloudPlatform/datashare-toolkit/blob/master/api/v1alpha/src/accounts/index.js#L645, but that needs to be updated too, so I'll create a bug.